### PR TITLE
fix #33568 borfilm.com

### LINKED ISSUE #33568: borfilm.com
### Issue URL (Ads)
[http://borfilm.com/25357-favoritka.html](https://adguardteam.github.io/AnonymousRedirect/redirect.html?url=http%3A%2F%2Fborfilm.com%2F25357-favoritka.html)
### Comment
Username: @big-u-boss
> 

### Screenshots
<details>
  <summary>Screenshot 1</summary>

  Screenshot 1
</details>

### System configuration

Information | value
--- | ---
Platform: | Ext
AdGuard version: | 3.0.14
Browser: | Chrome
Stealth mode options: | Hide your search queries, Remove X-Client-Data header from HTTP requests, Strip URLs from tracking parameters, Hide your Referrer from third-parties
Filters: | AdGuard Russian,<br>AdGuard Annoyances,<br>AdGuard Base,<br>AdGuard Tracking Protection,<br>AdGuard Social Media

### PR DESCRIPTION
**Pull request by zloyden**
https://github.com/AdguardTeam/AdguardFilters/issues/33568

### CODE REVIEW

**Review by BlazDT — COMMENTED** [review]
Unsure about this one, but looks strange:

https://send-notice.com/c/pop.php?sid=24067&bid=23

File: AnnoyancesFilter/sections/popups.txt [hunk 1]
```diff
@@ -93,6 +93,7 @@ upscpdf.com###fancybox_ns-overlay
 upscpdf.com###fancybox_ns-wrap
 bodrumluculuk.com##.ad-popup
 bodrumluculuk.com##.ad-popup-overlay
+borfilm.com#$#iframe[style*="display: block !important;"] { visibility: hidden!important; }
```
Comment (Alex-302) [inline AnnoyancesFilter/sections/popups.txt]: read BlazDT's comment. This is caused by third-party script(push notification)

**Review by Alex-302 — CHANGES_REQUESTED** [review]

File: AnnoyancesFilter/sections/popups.txt (hunk 1 above)
Comment (zloyden) [reply AnnoyancesFilter/sections/popups.txt]: Added push notification.

**Review by Alex-302 — APPROVED** [review]

File: SpywareFilter/sections/specific.txt [hunk 2]
```diff
@@ -11,6 +11,7 @@
 ! ||api.radio.*/info/*/user/playlog - breaks playing radio
 !
 ||linsoo.co.kr/linsoo_statistics_visitor.php
+borfilm.com##ul.right-bar
```
Comment (Alex-302) [inline SpywareFilter/sections/specific.txt]: image

**Review by Alex-302 — CHANGES_REQUESTED** [review]

File: SpywareFilter/sections/specific.txt (hunk 2 above)
Comment (zloyden) [reply SpywareFilter/sections/specific.txt]: Added domain to old rule with `##.count`.

**Review by Alex-302 — APPROVED** [review]

**Pull request merged**